src/watertap_contrib/seto/unit_models/surrogate/tests/test_med_tvc_surrogate.py

```diff
@@ -0,0 +1,340 @@
+        m = MED_TVC_frame
+        assert degrees_of_freedom(m) == 0
+
+    @pytest.mark.unit
+    def test_calculate_scaling(self, MED_TVC_frame):
+        m = MED_TVC_frame
+        calculate_scaling_factors(m)
+
+        # check that all variables have scaling factors
+        unscaled_var_list = list(unscaled_variables_generator(m))
+        assert len(unscaled_var_list) == 0
+
+        # check that all constraints have been scaled
+        unscaled_constraint_list = list(unscaled_constraints_generator(m))
+        assert len(unscaled_constraint_list) == 0
+
+    @pytest.mark.component
+    def test_var_scaling(self, MED_TVC_frame):
+        m = MED_TVC_frame
+        badly_scaled_var_lst = list(badly_scaled_var_generator(m))
+        assert badly_scaled_var_lst == []
+
+    @pytest.mark.component
+    def test_initialize(self, MED_TVC_frame):
+        m = MED_TVC_frame
+        initialization_tester(m, unit=m.fs.med_tvc, outlvl=idaeslog.DEBUG)
+
+    @pytest.mark.component
+    def test_solve(self, MED_TVC_frame):
+        m = MED_TVC_frame
+        results = solver.solve(m)
+
+        # Check for optimal solution
+        assert_optimal_termination(results)
+
+    @pytest.mark.component
+    def test_mass_balance(self, MED_TVC_frame):
+        m = MED_TVC_frame
+
+        med_tvc = m.fs.med_tvc
+
+        feed_flow_m3_hr = 277.777
+        dist_flow_m3_hr = 83.333
+        brine_flow_m3_hr = 194.444
+        cool_flow_m3_hr = 131.593
+
+        feed_mass_flow_tot = 78.93
+        cool_mass_flow_tot = 37.31
+        feed_mass_flow_tds = 2.70
+        brine_mass_flow_tds = 2.70
+        recovery = dist_flow_m3_hr / feed_flow_m3_hr
+
+        assert value(med_tvc.recovery_vol_phase[0, "Liq"]) == pytest.approx(
+            recovery, rel=1e-3
+        )
+        assert value(
+            pyunits.convert(
+                med_tvc.feed_props[0].flow_vol_phase["Liq"]
+                - med_tvc.distillate_props[0].flow_vol_phase["Liq"]
+                - med_tvc.brine_props[0].flow_vol_phase["Liq"],
+                to_units=pyunits.m**3 / pyunits.hr,
+            )
+        ) == pytest.approx(
+            feed_flow_m3_hr - dist_flow_m3_hr - brine_flow_m3_hr, rel=1e-3
+        )
+        assert value(med_tvc.feed_cool_mass_flow) == pytest.approx(
+            feed_mass_flow_tot + cool_mass_flow_tot, rel=1e-2
+        )  # mass flow calculated two different ways
+        assert value(med_tvc.feed_cool_vol_flow) == pytest.approx(
+            (feed_flow_m3_hr + cool_flow_m3_hr), rel=1e-3
+        )
+        assert value(
+            med_tvc.brine_props[0].flow_mass_phase_comp["Liq", "TDS"]
+            - med_tvc.feed_props[0].flow_mass_phase_comp["Liq", "TDS"]
+        ) == pytest.approx(feed_mass_flow_tds - brine_mass_flow_tds, rel=1e-6)
+
+        assert value(
+            med_tvc.brine_props[0].flow_mass_phase_comp["Liq", "TDS"]
+            - med_tvc.feed_props[0].flow_mass_phase_comp["Liq", "TDS"]
+        ) == pytest.approx(0, rel=1e-3)
```

Comment:
```suggestion
        ) == pytest.approx(0, abs=1e-3)
```
When checking for a 0 value, should use absolute tolerance instead of relative tolerance.

Reply:
Ah sure, it makes sense.